mffpy/tests/test_writer.py

```diff
@@ -74,18 +73,6 @@ def test_writer_writes(tmpdir):
     layout = R.directory.filepointer('sensorLayout')
     layout = XML.from_file(layout)
     assert layout.name == device
-    # cleanup
-    try:
-        remove(join(dirname, 'info.xml'))
-        remove(join(dirname, 'info1.xml'))
-        remove(join(dirname, 'epochs.xml'))
-        remove(join(dirname, 'signal1.bin'))
-        remove(join(dirname, 'coordinates.xml'))
-        remove(join(dirname, 'sensorLayout.xml'))
-        rmdir(dirname)
-    except BaseException:
-        raise AssertionError(f"""
-        Clean-up failed of '{dirname}'.  Were additional files written?""")
```

Comment:
This I kept in because I thought it was nice to check to make the exact set of files was written, but I suppose there are cleaner ways to do that